src/m365/planner/commands/plan/plan-get.ts

```diff
@@ -0,0 +1,160 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  title?: string;
+  ownerGroupId?: string;
+  ownerGroupName?: string;
+}
+
+class PlannerPlanGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.PLAN_GET;
+  }
+
+  public get description(): string {
+    return 'Get a Microsoft Planner plan';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = typeof args.options.id !== 'undefined';
+    telemetryProps.title = typeof args.options.title !== 'undefined';
+    telemetryProps.ownerGroupId = typeof args.options.ownerGroupId !== 'undefined';
+    telemetryProps.ownerGroupName = typeof args.options.ownerGroupName !== 'undefined';
+    return telemetryProps;
+  }
+
+  public defaultProperties(): string[] | undefined {
```

Comment:
We shouldn't define defaultProperties for a get command that returns a single item because it will unnecessarily trim the output

Reply:
The  `created by` property is also trimmed in other commands. Should we leave it, because it returns an object?
![image](https://user-images.githubusercontent.com/36161889/126161992-90b5753f-f0ea-48be-93ce-297f5c2c5eb4.png)

Reply:
Good catch. In that case, let's keep it consistent and trim it here as well 👍